site/security_users_groups_roles.md

```diff
@@ -51,3 +51,4 @@ topology:
 - The processing of users, groups, and roles will only take place when using the [Create Domain Tool](create.md)
 - WebLogic global roles are only supported with WebLogic Server version 12.2.1 or greater
 - WebLogic global roles are only updated for the WebLogic security XACML role mapping provider (i.e. `XACMLRoleMapper`)
+- The user and group processing is not complete, groups are created but users are only assigned to the `Administrators` group
```

Comment:
Please change this line to include "due to a bug in WLST".  Something like, "Due to a bug in WLST, Users cannot be assigned to groups.  Users created this way are automatically added to the Administrators group and are not added to the group(s) specified. As soon as a patch is available for WLST, we will post it here."

Reply:
As far as I know, we've never used the wording, "Due to a bug, ..." If it's important to you to state that, ok. Or, an alternative wording to consider is, " Currently, users cannot be assigned to groups. Users created this way (using WLST?) are automatically added to the Administrators group and are not added to the group(s) specified. As soon as a patch to correct this behavior is available, we will post it here."

Reply:
I have made a update to the wording based on Rosemary's suggested text so I think we should be good now. Thanks.